controller/controller.go

```diff
@@ -20,6 +20,10 @@ import (
 	"time"
 )
 
+// PDFjsExtensionID will be used to add an exception to pass the request for
+// PDF viewer extension install from chrome web store to the extension updater proxy
+var PDFjsExtensionID = "oemmndcbldboiebfnladdacbdfmadadm"
```

Comment:
minor: should probably be `PDFJS` not `PDFjs` for consistency

Reply:
resolved